docs/usage-examples/runCommand.txt

```diff
@@ -0,0 +1,52 @@
+.. _laravel-run-command-usage:
+
+=============
+Run a Command
+=============
+
+You can run a command directly on a database by calling the ``command()``
```

Comment:
Suggestion:

I think it could be helpful to specify that it's a MongoDB command to avoid confusion, similar to how it's used in the admonition at the end of the page.

```suggestion
You can run a MongoDB command directly on a database by calling the ``command()``
```